javascript-assignment/odd-even.js

```diff
@@ -0,0 +1,4 @@
+function isOdd(e) {
+    return false
+}
+isOdd(8)
```

Comment:
@HarishBoke Can you please add an extra line to the end of the file?

Reply:
Sure, I will update